### ISSUE TITLE
[Feature Request]: Implement Differential Diffusion

### ISSUE THREAD

**Issue by DanPli** [Panchovix/stable-diffusion-webui-reForge#107]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

Please consider adding Differential Diffusion. The ComfyUI implementation is [here ](https://github.com/comfyanonymous/ComfyUI/pull/2876) . This is absolutely amazing for inpainting, and it just works without bombarding the user with an array of settings.

### Proposed workflow

A simple checkbox in the inpaint tab (everywhere where masking happens) should suffice, as there are no settings to configure. A tooltip should explain that it modifies the mask behavior and that more steps give potentially finer results and perhaps a lower cfg should be considered if it does not work as intended (see comfy PR).

**Comment by Panchovix** [comment]
Hi there, thanks for the suggestion. This seems pretty applicable, I will take a look.

**Comment by Panchovix** [comment]
Okay did an small port

https://github.com/Panchovix/stable-diffusion-webui-reForge/commit/43414521b174e0861817ea7d2d8a3852fc302408

It doesn't seem to error out, but not sure how to use or see if it works, if you want to check.

**Comment by DanPli** [comment]
> Okay did an small port
> 
> [4341452](https://github.com/Panchovix/stable-diffusion-webui-reForge/commit/43414521b174e0861817ea7d2d8a3852fc302408)
> 
> It doesn't seem to error out, but not sure how to use or see if it works, if you want to check.

I just tested it and it seems to work just like in comfy.  I disabled "soft inpainting" to make sure there is interference going on there and just masked an extra leg I conveniently got on a quick gen to try it :)
A giveaway is that it adheres more to the original input (due to how it works), so on denoise 1.0 with fill option it would still recreate that extra leg, but when DD is disabled it replaced the masked area entirely.
DD is amazing to fix details like hair, eyes, faces... or generally speaking objects in the image that you want to keep but change their appearance. As a sideeffect it also should not produce an "inpaint halo" or seams ever.

So as far as I can tell, I would cautiously say this was a complete success, Thank you :)
